assignments/week_1/exercise_1_1/exercise_1_1_answer.cpp

```diff
@@ -0,0 +1,62 @@
+//Do not edit the next 5 lines
+#include <stdlib.h>
+#include <iostream>
+#include <vector>
+
+int main(int argc, char** argv) {
+    std::cout << "Use the wasd keys and enter to input a direction, or the E key to stop" << std::endl;
+
+    // Write your code for step 1 here
+    std::vector<float> wheel_speeds(4);
+    // Write your code for step 2 here
+    wheel_speeds[0] = 0;
+    wheel_speeds[1] = 0;
+    wheel_speeds[2] = 0;
+    wheel_speeds[3] = 0;
+    // Do not edit the next 5 lines
+    bool keep_going = true;
+    char direction;
```

Comment:
I know we're confident in this example that `direction` is assigned to before we use it, but I feel like we should initialize it to help enforce that habit.
```suggestion
    char direction = 0;
```